styles.css

```diff
@@ -27,7 +26,12 @@
     flex-basis: calc(100% - 4px);
 }
 
-.focus-mode
-    .side-dock-ribbon-action[aria-label="Toggle Focus Mode (Shift + Click to show active pane only)"] {
-    visibility: visible;
+.focus-mode div.workspace-ribbon.mod-left {
+    display: block;
+    position: absolute !important;
```

Comment:
Can we achieve this with a more specific selector without using `!important`?

Reply:
Some themes may change this property themselves

Reply:
Yeah, I think we should make it easy to override without needing obscene specificity.